tests/suites/test_suite_psa_crypto.function

```diff
@@ -6813,6 +7215,295 @@ exit:
+                                       signature, signature_length ) );
+
+    TEST_EQUAL( psa_verify_hash_start( &verify_operation, key, alg,
+                                       input_data->x, input_data->len,
+                                       signature, signature_length ),
+                PSA_ERROR_BAD_STATE );
+
+    PSA_ASSERT( psa_verify_hash_abort( &verify_operation ) );
+
+    /* --- Ensure changing the max ops mid operation works (operation should
+     *     complete successfully after setting max ops to unlimited --- */
+    psa_interruptible_set_max_ops( 1 );
+
+    PSA_ASSERT( psa_sign_hash_start( &sign_operation, key, alg,
+                                     input_data->x, input_data->len ) );
+
+    TEST_EQUAL( psa_sign_hash_complete( &sign_operation, signature,
+                                        signature_size,
+                                        &signature_length ),
+                PSA_OPERATION_INCOMPLETE );
+
+    psa_interruptible_set_max_ops( PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED );
+
+    PSA_ASSERT( psa_sign_hash_complete( &sign_operation, signature,
+                                        signature_size,
+                                        &signature_length ) );
+
+    PSA_ASSERT( psa_sign_hash_abort( &sign_operation ) );
+
+    psa_interruptible_set_max_ops( 1 );
+
+    PSA_ASSERT( psa_verify_hash_start( &verify_operation, key, alg,
+                                       input_data->x, input_data->len,
+                                       signature, signature_length ) );
+
+    TEST_EQUAL( psa_verify_hash_complete( &verify_operation ),
+                PSA_OPERATION_INCOMPLETE );
+
+    psa_interruptible_set_max_ops( PSA_INTERRUPTIBLE_MAX_OPS_UNLIMITED );
+
+    PSA_ASSERT( psa_verify_hash_complete( &verify_operation ) );
+
+    PSA_ASSERT( psa_verify_hash_abort( &verify_operation ) );
+
+    /* --- Change function inputs mid run, to cause an error (sign only,
+     *     verify passes all inputs to start. --- */
+
+    psa_interruptible_set_max_ops( 1 );
+
+    PSA_ASSERT( psa_sign_hash_start( &sign_operation, key, alg,
+                                     input_data->x, input_data->len ) );
+
+    TEST_EQUAL( psa_sign_hash_complete( &sign_operation, signature,
+                                        signature_size,
+                                        &signature_length ),
+                PSA_OPERATION_INCOMPLETE );
+
+    TEST_EQUAL( psa_sign_hash_complete( &sign_operation, signature,
+                                        0,
+                                        &signature_length ),
+                PSA_ERROR_BUFFER_TOO_SMALL );
+
+    PSA_ASSERT( psa_sign_hash_abort( &sign_operation ) );
+
+exit:
+    /*
+     * Key attributes may have been returned by psa_get_key_attributes()
+     * thus reset them as required.
+     */
+    psa_reset_key_attributes( &attributes );
+
+    psa_destroy_key( key );
+    mbedtls_free( signature );
+    PSA_DONE( );
+}
+/* END_CASE */
+#endif
```

Comment:
Where is the `#if` that goes with this `#endif`? When I grep for each there are 14 `#if`s and 15 `#endif`s that's what Uncrustify is complaining about. I guess I'm missing something as I'd expect the other tests to fail.

Reply:
Come to think of it this might be removed during code generation, which would be why none of the tests are failing, but it causes this file to be parsed wrongly by Uncrustify.

Reply:
Indeed, test code generation ignores stuff between `END_CASE` and `BEGIN_CASE`. But we should not rely on this permissiveness.